.github/workflows/ci.yaml

```diff
@@ -174,7 +174,7 @@ jobs:
 
     strategy:
       matrix:
-        k8s: [v1.19.11, v1.20.7, v1.21.1]
+        k8s: [v1.19.11, v1.20.7, v1.21.1, v1.21.2, v1.22.0]
```

Comment:
We can keep only one version of v1.21
(remove v1.21.1

Reply:
removed. done